src/background.js

```diff
@@ -99,6 +90,22 @@ function contextListener (info) {
   }
 }
 
+// listen to incoming messages from the content script and on new tab creation
+async function connectToActiveTab () {
+  let tab = await browser.tabs.query({ active: true, currentWindow: true })
+  tab = tab[0]
+  const tabName = `tab-${tab.id}`
+
+  if (contentPort) {
+    contentPort.onMessage.removeListener(api)
+    contentPort.disconnect()
+    contentPort = null
+  }
+
+  contentPort = browser.tabs.connect(tab.id, { name: tabName })
+  contentPort.onMessage.addListener(api)
```

Comment:
need to update currentPort to newly created contentPort if we still want content script to be able to call getCustomWords (which it currently does not) - the best time to do this would actually be once the disconnect event fires for the browser popup